docs/local-running.md

```diff
@@ -26,11 +26,27 @@ You will need PostgreSQL 9.+, Python 3.4+ and PIP installed.
   ```
   $ alembic --config config/alembic.ini upgrade head
   ```
+  
+5. Generate RSA key pair to use for database deployment signing:
+  ```
+  $ ssh-keygen \
```

Comment:
Multiline is great for readability, but not so for copy-paste. BTW, full (long) parameter names can explain things better.